admin.rst

```diff
@@ -141,6 +141,40 @@ Server Version
 
 When debugging a problem it's important to verify the PostgREST version. At any time you can make a request to the running server and determine exactly which version is deployed. Look for the :code:`Server` HTTP response header, which contains the version number.
 
+.. _error_source:
+
+Error Source
+------------
+
+For the most part, error messages will come directly from the database with the usual code and message that PostgreSQL uses. PostgREST will then convert the error to JSON format with an HTTP status code (see the full list :ref:`here <status_codes>`). For instance, this is the error you will get when querying a nonexistent table:
```

Comment:
How about if we mention that the json response corresponds to pg error message format (MESSAGE, DETAIL, HINT
, ERRCODE): https://www.postgresql.org/docs/12/plpgsql-errors-and-messages.html

Actually we don't use ERRCODE, but we can emphasize that we stick to our structure for all error bodies and that `details` and `hint` can be null.

Reply:
The above just made me notice something.

Do PostgreSQL errors always come with `hint` and `details` as `null`?

Because in our case, when we don't have `hint`/`details` we simply omit the keys instead of having them as null, right?

Perhaps we should change that for consistency?

Reply:
>  * We'll use five-character error codes to keep some consistency with PostgreSQL (`"code": "PostgREST"` is now `"code": "PGRST"`)
> 
>  * Other errors coming from Hasql will have their own codes e.g. `PGCON` for `ConnectionError`

This is no longer valid, we continued the discussion on the PR that implements this change: https://github.com/PostgREST/postgrest/pull/1926#issuecomment-911695893